packages/http-framework/src/lib/Client.ts

```diff
@@ -151,6 +153,11 @@ export interface ClientOptions {
 	 * @default `1024 * 1024` (1 MiB)
 	 */
 	bodySizeLimit?: number;
+	/**
+	 * Whether to suppress automatically sending a 500 status to Discord if errors occur in an interaction.
```

Comment:
```suggestion
	 * Whether to send a 500 status to Discord if an error occurs while processing an interaction.
```

Reply:
That changes the meaning of the sentence. The option is `true` if the error is suppressed, i.e. no 500 is sent.
Perhaps I could rename the variable to `handleErrors` or something so it's not a negating statement?

Reply:
Yes, please. I think a more fitting name would be `replyOnError` or maybe `httpReplyOnError`. 🤔